docs/components/modules/loader.md

```diff
@@ -38,6 +38,21 @@ This allows the replacement of applets, system modules, or even games with homeb
 ##### File Stubbing
 In order to prevent an NSO from being loaded even if it exists in the exefs, loader will also check if a stub file exists. If such a file exists, the NSO will not be loaded. The files should be named like `rtld.stub`, `main.stub`, etc. and may be empty.
 
+##### Technical Semantics
+
+loader's semantics for content override can (as you may observe from reading the above) be complicated to understand. The following is an abbreviated description of the very technical semantics by which loader decides what content to read when trying to read a file for a program id.
+
+* If an external content filesystem exists for the program id, the external content filesystem is used directly with no further redirection.
+* Otherwise, if the program ID is being overridden with [nx-hbloader](https://github.com/switchbrew/nx-hbloader/releases) (see Homebrew Support below), the nsp filesystem for hbl is used directly with no further redirection.
+* Otherwise, if content redirection is enabled for the program ID (controlled by a configurable button combination) and a loose file exists on the SD card, the loose file is used.
```

Comment:
Missing entire ExeFS replacement from `exefs.nsp`?

Reply:
Mentioned. I forget this is a thing because the new bindings only ever open that filesystem *or* the base filesystem (exception the real-base-filesystem-used-for-verif thing).